kazoo/tests/test_client.py

```diff
@@ -1158,6 +1164,147 @@ def test_request_queuing_session_expired(self):
         finally:
             client.stop()
 
+    def _require_zk_version(self, major, minor):
```

Comment:
This would benefit from a docstring describing the goal of the function... since the `pytest.skip` marker isn't until the end of the method, stating at the top "this function skips the test if the zookeeper version isn't greater than or equal to major.minor" would allow a dev to skim it faster.